lib/morph/database.rb

```diff
@@ -163,16 +163,16 @@ def write_sqlite_database(content)
     end
 
     def standardise_table_name(table_name)
-      sql_query_safe("ALTER TABLE '#{table_name}' " \
-        "RENAME TO '#{Database.sqlite_table_name}'", false)
+      sql_query_safe("ALTER TABLE \"#{table_name}\" " \
+        "RENAME TO \"#{Database.sqlite_table_name}\"", false)
     end
 
     def select_first_ten(table = table_names.first)
-      "select * from '#{table}' limit 10"
+      "select * from \"#{table}\" limit 10"
```

Comment:
Let's be clear: I'm proposing this solely because it's a moderately-serious UI fail - where this query is reprinted in the API documentation it *encourages* users to use single quotes, the first thing I tried was to add an ORDER BY to the example, and with single quotes that query outright fails to work.

So yeah, by all means, but this isn't rooted in me being anal about SQL style, but about whether the API works as documented. Right now, it (slightly) does not.